mods/ascent/datum/culture.dm

```diff
@@ -80,9 +77,9 @@ GLOBAL_LIST_INIT(gyne_architecture, list(
 
 /decl/cultural_info/culture/ascent/get_random_name(var/gender)
 	if(gender == MALE)
-		return "[random_id(/datum/species/mantid, 10000, 99999)] [get_gyne_name()]"
+		return "[random_id(/datum/species/mantid, 10000, 99999)] [create_gyne_name()]"
```

Comment:
This will make the Ascent submap have a bunch of Kharmaani with different names instead of sharing a lineage. Either need to update the submap join code or use a different method for passing on lineage name. Preferrably lineage would be used by submap code as well.